src/api/v1/endpoint/chat.py

```
@@ -27,21 +24,17 @@ def chat_endpoint_test(request: ChatRequest):
 
 
 @router.post("/chat", response_model=CommonResponse)
-async def chat_endpoint(
-    request: ChatRequest, db: AsyncSession = Depends(get_db)  # 👈 DB 의존성 주입
-):
+async def chat_endpoint(request: ChatRequest, db: AsyncSession = Depends(get_db)):
```

Ruff (0.15.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)